src/backend/common/queries/team_query.py

```diff
@@ -94,10 +95,13 @@ def __init__(self, district_key: DistrictKey) -> None:
     def _query_async(
         self, district_key: DistrictKey
     ) -> Generator[Any, Any, List[Team]]:
+        year = int(re.match(r"\d{4,}", district_key).group(0))
```

Comment:
I'm assuming there's some sort of helper for parsing these but I couldn't find one.